docs/phase_0.md

```diff
@@ -4,78 +4,149 @@ _Warning_: This a work in progress document with many broken links, some unfinis
 
 ## Introduction
 
-This document is an aid to the specification of the [state transition function](https://github.com/ethereum/eth2.0-specs/blob/v0.10.0/specs/phase0/beacon-chain.md#beacon-chain-state-transition-function) and accompanying [data structures](https://github.com/ethereum/eth2.0-specs/blob/v0.10.0/specs/phase0/beacon-chain.md#containers) for Phase 0 - The Beacon Chain -- the core system level chain at the heart of Ethereum 2.0. The [primary Phase 0 specification](https://github.com/ethereum/eth2.0-specs/blob/v0.10.0/specs/phase0/beacon-chain.md) is particularly terse, only defining much of the functionality via python code and comments. This document aims to provide a more verbose version of the specification, especially aimed at helping onboard new contributors to Ethereum 2.0.
+This document is an aid to the specification of the [state transition function](https://github.com/ethereum/eth2.0-specs/blob/v0.10.0/specs/phase0/beacon-chain.md#beacon-chain-state-transition-function) and accompanying [data structures](https://github.com/ethereum/eth2.0-specs/blob/v0.10.0/specs/phase0/beacon-chain.md#containers) for Phase 0 - The Beacon Chain -- the core system level chain at the heart of Ethereum 2.0. 
+
+The [primary Phase 0 specification](https://github.com/ethereum/eth2.0-specs/blob/v0.10.0/specs/phase0/beacon-chain.md) is particularly terse, only defining much of the functionality via python code and comments. This document aims to provide a more verbose version of the specification, primarily aimed at helping onboard new contributors to Ethereum 2.0.
 
 ## Table of contents
 
 [toc]
 
 ## High level overview
 
-The beacon chain is the core system-level blockchain for Ethereum 2.0. The name comes from that fact that it serves as a random beacon, but could just as easily be called the "system chain", "spine chain", etc. This chain stores and manages the registry of validators in which the validators are assigned their _duties_: perform a protocol level _RNG_, _progress the beacon chain_, _vote on the head_ of the chain for the fork choice, _finalize checkpoints_, and _link and vote in transitions/data of shard chains_. The beacon chain is both the brains behind the operation and the scaffolding upon which the rest of the sharded system is built.
+The beacon chain is the core system-level blockchain for Ethereum 2.0. The name has its origins in the idea of a [random beacon](https://csrc.nist.gov/projects/interoperable-randomness-beacons)-- a beacon that provides a source of randomness to the rest of the system -- but it could just as easily have been called the "system chain", "spine chain", or something similar.
+
+A major part of the work of the beacon chain is storing and managing the registry of validators -- the set of nodes that have placed the required stake of 32 Ether, and who are responsible for running the Ethereum 2.0 system.
```

Comment:
validators are not _necessarily_ 1:1 with nodes. Each 32 eth buys you a validator position in the consensus but many validators can be run on a single _node_.

So maybe "the set of participants" or "the set of consensus participants"